source/includes/many-with-one-cluster.rst

```diff
@@ -0,0 +1,3 @@
+Starting in ``mongosync`` 1.7.0, you can sync multiple source clusters
+simultaneously with a destination cluster. For example, consolidate data
+from many small clusters into one central cluster.
```

Comment:
```suggestion
simultaneously with a destination cluster. For example, you can consolidate data
from many small clusters into one central cluster.
```

Reply:
Done. Though generally, we don't like to repeat the "you can" twice in the same para.